db/seeds.rb

```diff
@@ -2,43 +2,9 @@
 
 return if Rails.env.production?
 
-# NOTE: when adding new data, please use this class to ensure the seed tasks
+# NOTE: when adding new data, please use the Seeder class to ensure the seed tasks
 # stays idempotent.
-class Seeder
```

Comment:
As mentioned above, I pulled out the `Seeder` class so that it could be reused in my e2e data seeding.